CommonClient.py

```diff
@@ -70,59 +70,120 @@ def _cmd_disconnect(self) -> bool:
             if location_id < 0:
                 continue
             if location_id not in self.ctx.locations_checked:
                 if location_id in self.ctx.missing_locations:
                     self.output('Missing: ' + location)
                     count += 1
+                    total_count += 1
+                elif location_id in self.ctx.checked_locations:
+                    total_count += 1
+
+        if count:
+            self.output(f"Found {count}/{total_count} {'matching ' if filter_text else ''}missing checks")
+        else:
+            self.output("No missing checks found")
+        return True
+
+    @mark_raw
+    def _cmd_checked(self, filter_text: str = "") -> bool:
+        """List all checked location checks from your local game state.
+        Can be given text, which will be used as filter."""
+        if not self.ctx.game:
+            self.output("No game set, cannot determine checked checks.")
+            return False
+        if filter_text:
+            self.output(f"Checked locations matching '{filter_text}':")
+        else:
+            self.output("Checked locations:")
+        count = 0
+        total_count = 0
+        for location, location_id in AutoWorldRegister.world_types[self.ctx.game].location_name_to_id.items():
+            if filter_text and filter_text.lower() not in location.lower():
+                continue
+            if location_id < 0:
+                continue
+            if location_id not in self.ctx.locations_checked:
+                if location_id in self.ctx.missing_locations:
+                    total_count += 1
                 elif location_id in self.ctx.checked_locations:
                     self.output('Checked: ' + location)
                     count += 1
-                    checked_count += 1
+                    total_count += 1
 
         if count:
-            self.output(
-                f"Found {count} missing location checks{f'. {checked_count} location checks previously visited.' if checked_count else ''}")
+            self.output(f"Found {count}/{total_count} {'matching ' if filter_text else ''}checked locations")
         else:
-            self.output("No missing location checks found.")
+            self.output("No checked locations found")
         return True
 
-    def _cmd_items(self):
-        """List all item names for the currently running game."""
+    @mark_raw
+    def _cmd_items(self, filter_text: str = ""):
+        """List all item names for the currently running game.
+        Can be given text, which will be used as filter."""
         if not self.ctx.game:
             self.output("No game set, cannot determine existing items.")
             return False
-        self.output(f"Item Names for {self.ctx.game}")
+        if filter_text:
+            self.output(f"Item Names for {self.ctx.game} matching '{filter_text}':")
+        else:
+            self.output(f"Item Names for {self.ctx.game}:")
+        self.output("")
```

Comment:
Does this have a blank line here, while the similar locations command doesn't?

Reply:
...yep, nice catch. Wanted that on both, for spacing.